views/index.ejs

```diff
@@ -212,6 +214,7 @@
 <input type="hidden" id="user-id" value= <%= user?user.uid:"visitor" %> />
 <input type="hidden" id="user-name" value= <%= user?user.cn:"" %> />
 <input type="hidden" id="user-email" value= <%= user?user.mail:"" %> />
+<input type="hidden" id="user-admin" value= <%= user?(user.admin?"true":"false"):"false" %> />
```

Comment:
Note that this field is only to aid the UI, but the validator client-side doesn't add any security.

Reply:
Yup.. there isn't any validator on the client side, so the backend must catch bad requests. What change would you suggest?